node/resolvers/Mutations/Users.ts

```diff
@@ -300,13 +299,14 @@ const Users = {
           ) &&
             !roleSlug.includes('admin') &&
```

Comment:
Are we keeping the "admin" role check here? Basically we are saying that, if this user has the `impersonate-users-costcenter` permission but is an `admin`, it can't impersonate, which doesn't make sense I think...

Reply:
I forgot. Thanks. I changed `impersonateB2BUser`, but I will change `impersonateUser` too.